python-client/tests/integrations/test_wandb.py

```diff
@@ -51,4 +69,25 @@ def _to_wandb(model, dataset):
     test_suite_results = scan_results.generate_test_suite().run()
     test_suite_results.to_wandb()
 
+    # Verify that the logging of the SHAP explanation charts works.
+    explanation_results = explain_with_shap(model, dataset)
+    explanation_results.to_wandb()
+
     assert re.match("^[0-9a-z]{8}$", str(wandb.run.id))
+
+
+def _compare_explain_functions(model, dataset):
```

Comment:
can you move this to `test_model_explanation.py`?

Reply:
Done